chromium/background.js

```diff
@@ -1,13 +1,7 @@
-/* exported enableMixedRulesets */
-/* global RuleSets */
-/* global storage */
-/* global log */
-/* global INFO */
-/* global NOTE */
-/* global WARN */
-/* global DBUG */
+(function(exports) {
 
 "use strict";
```

Comment:
This should probably have global scope, rather than function scope

Reply:
done, for other files too